pages/article_detail.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="page-wrapper">
+        <v-img
+            height="600"
+            lazy-src="https://via.placeholder.com/600"
+            :src="selectedArticle.urlToImage"
+        ></v-img>
+        <div class="pa-16">
+            <h1 class="page-heading">{{ selectedArticle.title || "--" }}</h1>
+            <div class="py-5 text-h6">
+                <div><span class="font-weight-light">Author</span>: {{ selectedArticle.author || "--" }}</div>
+                <div><span class="font-weight-light">Published On</span>: {{ formattedPublishDate || "--" }}</div>
+            </div>
+            <div>
+                <div class="text-subtitle-1">Article Detail:</div>
+                <p class="text-justify text-h6">{{ selectedArticle.content }}</p>
+            </div>
+            <div>
+                <p class="text-justify"><a :href="selectedArticle.url" target="_blank" rel="noopener noreferrer">Read Full Article</a></p>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+// Library imports
+import 'reflect-metadata'
+import { Vue, Component } from 'vue-property-decorator'
+
+// Utils and Interfaces imports
+import { ArticleObj } from '~/utils/interfaces'
+
+// Vuex imports
+import { mapMutations, mapState } from 'vuex'
+
+@Component({
+    methods: {
+        ...mapMutations({
+            setSelectedArticle: 'setSelectedArticle'
+        })
+    },
+    computed: {
+        ...mapState([
+            'selectedArticle'
+        ])
+    }
+})
+export default class ArticleDetail extends Vue {
+
+    $helper: any // To avoid Typescript errors related to type safety
+
+    // Props
+
+
+    // Component Data
+
+
+    //Computed Properties
+    protected selectedArticle!: ArticleObj // To avoid Typescript errors related to type safety
+
+    get formattedPublishDate() {
+        return this.selectedArticle ? this.$helper.formatPublishDate(this.selectedArticle.publishedAt) : ""
+    }
+
+    //All methods
+
+}
+</script>
+
+<style lang="scss" scoped>
+.page-heading {
+    color: $bio-indigo;
+}
```

Comment:
we prefer to define classes on .scss and use classes here,
like you could have define 
```scss
   .text-indigo{
   color:$bio-indigo;
   }
  ```
  and use it here directly
  
  and it's better to name property based not content based
  like `text-indigo` would be more better than `page-heading` what if you need same color text for other element.

Reply:
Yes. In real project I will add it in global file. For the practice project, it was mentioned to use scoped css. :sweat_smile: